README.md

```diff
@@ -193,6 +193,7 @@ You can pass the following environment variables to LocalStack:
 * `DYNAMODB_ERROR_PROBABILITY`: Decimal value between 0.0 (default) and 1.0 to randomly inject `ProvisionedThroughputExceededException` errors into DynamoDB API responses.
 * `DYNAMODB_HEAP_SIZE`: Sets the JAVA EE maximum memory size for dynamodb values are (integer)m for MB, (integer)G for GB default(256m), full table scans require more memory
 * `STEPFUNCTIONS_LAMBDA_ENDPOINT`: URL to use as the lambda service endpoint in step functions. By default this is the localstack lambda endpoint.
```

Comment:
Thanks for this pull request @dstasiuk ! I'd prefer if we can keep the number of configuration options as low as possible - especially if config options are mutually exclusive, as in this particular example.

Can we please remove the `STEPFUNCTIONS_EXCLUDE_LAMBDA_ENDPOINT` entirely, and introduce a special value (e.g., `"exclude"`) for the existing `STEPFUNCTIONS_LAMBDA_ENDPOINT` variable? I'd propose something like this:

>  * `STEPFUNCTIONS_LAMBDA_ENDPOINT`: URL to use as the lambda service endpoint in step functions. By default this is the localstack lambda endpoint. Set the value to `"exclude"` to skip adding the Lambda service endpoint to the StepFunctions configuration.

Reply:
Hi @whummer . Yeap, it makes sense. Fixed!